q2_sample_classifier/_format.py

```diff
@@ -163,3 +163,15 @@ def _validate_(self, level):
 ImportanceDirectoryFormat = model.SingleFileDirectoryFormat(
     'ImportanceDirectoryFormat', 'importance.tsv',
     ImportanceFormat)
+
+
+class ProbabilitiesFormat(ImportanceFormat):
```

Comment:
How about instead we create a new base class and have both of these formats inherit from it?

Reply:
👍